tests/authentication/test_authentication.py

```diff
@@ -0,0 +1,45 @@
+from allure import description, step, title, feature
+from hamcrest import assert_that, is_, is_not, empty
+
+from framework.endpoints.authenticate_api import AuthenticateAPI
+from framework.endpoints.users_api import UsersAPI
+from framework.tools.generator_random_data import generate_user_data
+
+
+@feature("Authentication of user")
+class TestAuthentication:
+    @title("Test authentication")
+    @description(
+        "WHEN user submit valid credential "
+        "THEN status HTTP CODE = 200 and get JWT token"
+    )
+    def test_authentication(self, postgres):
+        with step("Generation data for registration"):
+            data = generate_user_data(length_first_name=8, length_last_name=8, password_len=8)
+
+        with step("Registration new user"):
+            response = AuthenticateAPI().registration(body=data)
+            assert_that(response.status_code, is_(201))
```

Comment:
Please add an error message for each assertion. For example:
assert_that(
                response.status_code, is_(201), reason='Expected status code 201'
            )

Reply:
Resolved